skipruntime-ts/core/src/internals/skipruntime_module.ts

```diff
@@ -831,7 +831,7 @@ class LinksImpl implements Links {
   env: Environment;
   handles: Handles;
   skjson?: SKJSON;
-  timedQueue?: TimedQueue;
+  timedQueue?: ReturnType<typeof setInterval>[];
```

Comment:
```suggestion
  timedQueue?: ReturnType<NodeJS.Timeout>[];
```
You want the return type, not the function itself

Reply:
Hmm the `ReturnType<T>` should give the return type of the function type `T`. The reason I didn't go with `NodeJS.Timeout` is to preserve browser compatibility.